docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1465,6 +1465,24 @@ How to **investigate**:
 - Check if ingesters are processing too many records, and they need to be scaled up (vertically or horizontally).
 - Check actual error in logs to see whether the `-ingest-storage.kafka.wait-strong-read-consistency-timeout` or the request timeout has been hit first.
 
+### MimirKafkaClientBufferedProduceBytesTooHigh
+
+This alert fires when the Kafka client buffer, used to write incoming write requests to Kafka, is getting full.
+
+How it **works**:
+
+- Distributor and ruler encapsulate write requests into Kafka records and send them to Kafka.
+- The Kafka client has a limit on the total bytes size of records buffered to be sent to Kafka or sent to Kafka but not acknowledged yet.
+- When the limit is reached, Kafka client will reject to produce more records and it will fast fail.
+- The limit is configured via `-ingest-storage.kafka.producer-max-buffered-bytes`.
+- The default limit is configured intentionally high so that when the buffer utilization gets close to the limit then there's probably an issue.
```

Comment:
```suggestion
- The default limit is configured intentionally high, so that when the buffer utilization gets close to the limit, this indicates that there's probably an issue.
```